lib/elixir/test/elixir/tuple_test.exs

```diff
@@ -62,4 +62,11 @@ defmodule TupleTest do
     mod = Tuple
     assert mod.delete_at({:foo, :bar, :baz}, 0) == {:bar, :baz}
   end
+
+  test "sum/1" do
+    assert Tuple.sum({255, 255}) == 510
+
+    mod = Tuple
+    assert mod.sum({255, 255}) == 510
```

Comment:
This test is only needed if the function has rewrite optimizations which it does not have.